pkg/azureutils/azure_disk_utils.go

```diff
@@ -206,13 +206,15 @@ func CreateValidDiskName(volumeName string, usedForLabel bool) string {
 }
 
 // GetCloudProviderFromClient get Azure Cloud Provider
-func GetCloudProviderFromClient(kubeClient *clientset.Clientset, secretName string, secretNamespace string, userAgent string) (*azure.Cloud, error) {
```

Comment:
We shouldn't override our changes in `GetCloudProviderFromClient`. The changes from master should be ignored for this function.

Reply:
Well, except that we'll need to support the `allowEmptyCloudConfig` parameter, though I'm not sure why this was added in the first place. I'm fine with ignoring it for now as long as we file an issue to follow up on what the intended use case is for it.

Reply:
As @landreasyan suggested to keep the existing implementation of `GetCloudProviderFromClient`, I will remove the dependency on `allowEmptyCloudConfig` for now until we reach a resolution.